src/Core/Expressions/MemoryAccess.cs

```diff
@@ -125,19 +125,54 @@ public override T Accept<T>(ExpressionVisitor<T> visit)
             return visit.VisitSegmentedAccess(this);
         }
 
-		public override void Accept(IExpressionVisitor visit)
-		{
-			visit.VisitSegmentedAccess(this);
-		}
+        public override void Accept(IExpressionVisitor visit)
+        {
+            visit.VisitSegmentedAccess(this);
+        }
 
-		public override Expression CloneExpression()
-		{
-			return new SegmentedAccess(MemoryId, BasePointer.CloneExpression(), EffectiveAddress.CloneExpression(), DataType);
-		}
+        public override Expression CloneExpression()
+        {
+            return new SegmentedAccess(MemoryId, BasePointer.CloneExpression(), EffectiveAddress.CloneExpression(), DataType);
+        }
 
         public static SegmentedAccess Create(Expression segRegister, Expression baseRegister, int offset, DataType dt)
         {
             return new SegmentedAccess(MemoryIdentifier.GlobalMemory, segRegister, CreateEffectiveAddress(baseRegister, offset), dt);
         }
-	}
+    }
+
+#if false
+
+    /// <summary>
+    /// Models an access to data memory bank for 8-bit PIC.
+    /// </summary>
+    public class BankMemoryAccess : MemoryAccess
```

Comment:
@chostelet adding new expression class is the last resort. Consider using existing ones (e.g. `SegmentedAccess`). Have you discussed it with @uxmal?